From 148595ce8c62248841f945d0084f939eaf131473 Mon Sep 17 00:00:00 2001
From: Mikolaj Izdebski <mizdebsk@redhat.com>
Date: Tue, 20 Jan 2015 12:30:09 +0100
Subject: [PATCH 11/14] Add build dependency on ASM 5.0.3

In Fedora ASM was unbundled from Jar Jar, so it needs to be explicitly
added as dependency.
---
 buildSrc/build.gradle | 1 +
 1 file changed, 1 insertion(+)

diff --git a/buildSrc/build.gradle b/buildSrc/build.gradle
index 138b851..83a9006 100644
--- a/buildSrc/build.gradle
+++ b/buildSrc/build.gradle
@@ -41,6 +41,7 @@ dependencies {
     //it's built from sources at: https://github.com/szczepiq/jarjar
     //if code changes are needed in this library we need to figure out a better way of distributing them.
     compile "org.gradle.jarjar:jarjar:1.2.1"
+    compile "org.ow2.asm:asm-all:5.0.3"
 }
 apply from: '../gradle/compile.gradle'
 apply from: '../gradle/codeQuality.gradle'
-- 
2.1.0

